docs/.c4/model.c4

```diff
@@ -0,0 +1,248 @@
+model {
+  // personas
+  admin = actor "Cluster Administrator" {
+    admin -> external-admin-loadbalancer
+  }
+
+  enduser = actor "End User" {
+    enduser -> external-tenant-loadbalancer
+  }
+
+  infrastructure = infrastructure "Infrastructure" {
+
+    external-tenant-loadbalancer = loadbalancer "External Tenant LoadBalancer"
+    external-tenant-loadbalancer -> istio-tenant-ingress-gateway
+
+    external-admin-loadbalancer = loadbalancer "External Admin LoadBalancer"
+    external-admin-loadbalancer -> istio-admin-ingress-gateway
+
+    cluster = cluster "Kubernetes Cluster" {
+      // system
+      uds-core = system "UDS Core" {
+        description "UDS Core"
+
+        pepr-system = namespace "Pepr System Namespace" {
+          pepr-watcher = pod "Pepr Watcher"
+          pepr-admission = pod "Pepr Admission"
+
+          // external relationships
+          pepr-watcher -> kubernetes-control-plane
+          pepr-watcher -> kubernetes-control-plane.api-server
+          pepr-admission -> kubernetes-control-plane.pepr-mutation-webhook
+          pepr-admission -> kubernetes-control-plane.pepr-validating-webhook
+        }
+
+        zarf = namespace "Zarf Namespace" {
```

Comment:
Technically not _part of core_, but is required/integral for core. I could see moving it to the cluster layer or leaving in core system grouping.